ddl/index.go

```diff
@@ -1253,6 +1252,55 @@ func (w *baseIndexWorker) AddMetricInfo(cnt float64) {
 	w.metricCounter.Add(cnt)
 }
 
+func (*baseIndexWorker) GetTask() (*BackfillJob, error) {
+	return nil, nil
+}
+
+func (w *baseIndexWorker) String() string {
+	return w.tp.String()
+}
+
+func (w *baseIndexWorker) UpdateTask(bfJob *BackfillJob) error {
+	sess, ok := w.backfillCtx.sessCtx.(*session)
+	if !ok {
+		return errors.Errorf("sess ctx:%#v convert session failed", w.backfillCtx.sessCtx)
+	}
+
+	return runInTxn(sess, func(se *session) error {
+		jobs, err := GetBackfillJobs(sess, BackfillTable, fmt.Sprintf("ddl_job_id = %d and ele_id = %d and id = %d and ele_key = '%s'",
+			bfJob.JobID, bfJob.EleID, bfJob.ID, bfJob.EleKey), "update_backfill_task")
+		if err != nil {
+			return err
+		}
```

Comment:
Why do we need to get it first before updating it?

Reply:
In order to obtain error information, a direct update without updating data will not report an error.